app/models/manageiq/providers/hawkular/middleware_manager/event_catcher/runner.rb

```diff
@@ -1,20 +1,21 @@
 class ManageIQ::Providers::Hawkular::MiddlewareManager::EventCatcher::Runner <
   ManageIQ::Providers::BaseManager::EventCatcher::Runner
 
+  TAG_EVENT_TYPE    = "miq.event_type".freeze # required by fetch
```

Comment:
"tag" might be confusing as I assume it's not related to the tags feature of miq - unless I misunderstand what this code is doing